code/ExercisesCheatSheet.md

```diff
@@ -109,9 +109,10 @@ See in valgrind the improvements.
 
 This exercise has several levels. People not at ease can stop after first level and go to next exercise. Alternatively, they may do level 1 and 3 and skip 2.
 
-Level 1 : just use the given Complex class in OrderedVector and see it works out of the box thanks to generic code in OrderedVector.
+Level 1: just use the given `Complex` class in `OrderedVector` and see it works out of the box thanks to generic code in `OrderedVector`.
+Make sure, that students don't create a `Complex<std::string>`.
```

Comment:
Please explain why they might do that. I don't understand what I'm supposed to do as mentor. Imagine this:
"The next one is the templates exercise. Don't use `Complex<std::string>`, though!"

I think the students will be as confused as I am now.

Reply:
I have literally no idea. Around half of my students in the JLab lecture started to create `Complex<std::string>` and were struggling with compilation errors. I don't know where they picked up this idea. I am trying to massage the instructions to avoid future students to fall into this pit.

Reply:
I removed it again and massaged the other instrucitons a little bit and added a requires clause to `Complex<T>`.

Reply:
Maybe instead of saying:
Don't do this

we could say:
To do this, start by xxxx

Reply:
I extended the instructions and cheatsheet a bit.